src/main/scala/fi/oph/koski/mydata/ApiProxyServlet.scala

```diff
@@ -46,6 +54,23 @@ class ApiProxyServlet(implicit val application: KoskiApplication) extends ApiSer
       throw InvalidRequestException(KoskiErrorCategory.badRequest.header.invalidXRoadHeader))
       .getString("id")
   }
+
+  private def toMyDataOppijaResponse(oid: String, memberId: String, value: Either[HttpStatus, Oppija]): Either[HttpStatus, MyDataOppija] = {
+    value match {
```

Comment:
Oisko esim `oppija` parempi nimi tolle muuttujalle? Sitä tuolla käytetään noille Eithereille missä oppija on Right:in sisällä muuallakin.